src/lang/modifyAst.ts

```diff
@@ -443,6 +443,60 @@ export function loftSketches(
   }
 }
 
+export function addShell({
+  node,
+  sweepName,
+  faces,
+  thickness,
+  insertIndex,
+  variableName,
+}: {
```

Comment:
Moving the addShell function here to be more consistent with the other commands, and also taking the chance to change its scope a little bit, making this just about the creation of the new variable and new pathToNode buildup, not about all the selection prep logic, which is moved back to modelingMachine.ts' ast mod promise, like the other ones.